email_templates.py

```
@@ -216,3 +217,492 @@ def create_password_reset_email(reset_link, recipient_email):
+                    if matched_variant:
+                        extra = (
+                            f" (+PKR {float(matched_variant.price):.0f})"
+                            if float(matched_variant.price) > 0
+                            else ""
+                        )
+                        variant_lines.append(f"{name}: {value}{extra}")
+                    else:
+                        variant_lines.append(f"{name}: {value}")
+
+                variant_info = " (" + ", ".join(variant_lines) + ")"
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)